.nvmrc

```
@@ -1 +1 @@
-lts/boron
+10.16.0
```

I would suggest using an LTS alias, version 10 codename is `Dubnium`.

```suggestion
lts/dubnium
```